README.md

```diff
@@ -1 +1,36 @@
-# solarperformanceinsight
+# Solar Performance Insight
+
+Solar Performance Insight is an open source platform for evaluating solar power
+plant performance. Solar Performance Insight aims to provide an alternative
+to available asset management tools for PV Operations and Mainteneance (O&M)
+providers where existing tools are unaffordable or impractical.
```

Comment:
```suggestion
to available asset modeling tools for PV Operations and Mainteneance (O&M)
providers where existing tools are unaffordable or impractical.
```

Asset management is probably understood to encompass maintenance scheduling, parts ordering, billing, etc. all aspects of keeping a PV system operational and generating revenue. We only have modeling and analysis tasks in our scope.